app/models/facility.rb

```diff
@@ -0,0 +1,11 @@
+class Facility < ApplicationRecord
+  include Mergeable
+
+  has_many :facility_patients
+  has_many :patients, through: :facility_patients
+
+  validates :name, presence: true
+  validates :district, presence: true
+  validates :state, presence: true
+  validates :country, presence: true
```

Comment:
Since district, state and country are required fields, we should probably mark them required, and non-empty in the API spec as well.